docs/multi_file_content_management.md

```diff
@@ -0,0 +1,25 @@
+# Multi-file content management
+
+Multi-file content management is a set of functionalities that enables you to upload, move and delete multiple binary files in a bulk.
```

Comment:
It's not just binary files, but all Content items, right?
Also, `in bulk`, without article